### ISSUE TITLE
ICICI bank home page ! After login link next focusable element is not label and not activated

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#1]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2 after tab 15 time   reach to login link.
the element after login link the next element is not accessible 
 
Expected Result the element should be announce properly and also activated

Actual Result ‘clickable seven ’ announced by screen reader and also not activated after press enter
WCAG Guideline
https://www.w31/.  .org/TR/WCAG21/#keyboard
2.  https://www.w3.org/TR/WCAG21/#name-role-value
